src/mailing_info.cpp

```diff
@@ -58,8 +58,6 @@ auto lwg::make_html_anchor(lwg::issue const & iss) -> std::string {
    auto temp = std::to_string(iss.num);
 
    std::string result{"<a href=\""};
-   result += filename_for_status(iss.stat);
-   result += '#';
```

Comment:
On second thought, this part might not work too well with the open-std.org setup.